cmd/utils.go

```diff
@@ -946,3 +947,43 @@ func getWorkDirectory(containerFlavor string) string {
 func setWorkDirectory(value string) {
 	workingDirectory = value
 }
+
+// makeCephNanoPath is a utility to calculate a relative path to cephnano directory.
+func makeCephNanoPath(fileName ...string) string {
+	args := []string{getCephNanoPath()}
+	args = append(args, fileName...)
+	return filepath.Join(args...)
+}
+
+// getCephNanoPath returns the path to the user's cephnano directory
+func getCephNanoPath() string {
+	homeDirPath := ""
+	if runtime.GOOS == "windows" {
+		// First prefer the HOME environmental variable
+		if home := os.Getenv("HOME"); len(home) > 0 {
+			if _, err := os.Stat(home); err == nil {
+				homeDirPath = home
+			}
+		}
+		if homeDrive, homePath := os.Getenv("HOMEDRIVE"), os.Getenv("HOMEPATH"); len(homeDrive) > 0 && len(homePath) > 0 {
+			homeDir := homeDrive + homePath
+			if _, err := os.Stat(homeDir); err == nil {
+				homeDirPath = homeDir
+			}
+		}
+		if userProfile := os.Getenv("USERPROFILE"); len(userProfile) > 0 {
+			if _, err := os.Stat(userProfile); err == nil {
+				homeDirPath = userProfile
+			}
+		}
+	} else {
+		homeDirPath = os.Getenv("HOME")
```

Comment:
Please use these as a reference https://github.com/ceph/cn/blob/bdf9e3a8b78649699df1be07a06889704959d875/cmd/config_file.go#L79-L81 so we get consistent behavior from the existing config option.